### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by t-900-a**
skeleton, not working

### CODE REVIEW

**Comment by t-900-a** [comment]
https://codecov.io/gh/t-900-a/moneroodoo/tree/9c04d49daaad20612c797eb1916beb6c43280679
Travis ci build now passing
https://travis-ci.com/t-900-a/moneroodoo.svg?branch=main

File: monero-rpc-odoo/models/monero_acq.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        string="RPC Host",
+        help="The ip address or host name of the Monero RPC",
+        default="127.0.0.1",
+    )
+    monero_rpc_config_port = fields.Integer(
+        string="RPC Port",
+        help="The port the Monero RPC is listening on",
+        default=18082
+    )
+    monero_rpc_config_path = fields.Char(
```
Comment (serhack) [inline monero-rpc-odoo/models/monero_acq.py]: RPC path usually never change. Better to remove that from settings.

File: monero-rpc-odoo/data/currency.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <!--XMR Currency Record-->
+        <record id="currency_xmr" model="res.currency">
+            <field name="name">XMR</field>
+            <field name="currency_unit_label">Monero</field>
+            <field name="symbol">ɱ</field>
```
Comment (serhack) [inline monero-rpc-odoo/data/currency.xml:8]: Great 👍🏻

File: monero-rpc-odoo/controllers/controllers.py [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+        payment_token = {
+            "name":
+            str(source_wallet_address).strip() + " - " + kwargs.get("type"),
+            "partner_id": payment_partner_id,
+            # partner_id creating sales order
+            "active": True,
+            "acquirer_id": payment_acquirer_id,
+            # surrogate key for payment acquirer
+            "acquirer_ref": "cryptocurrency",
+            # this should be the tx hash?
```
Comment (serhack) [inline monero-rpc-odoo/controllers/controllers.py]: No! Never store something sensible as tx_hash!

File: monero-rpc-odoo/README.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+[![Build Status](https://travis-ci.com/t-900-a/moneroodoo.svg?branch=master)](https://travis-ci.com/t-900-a/moneroodoo)
+[![codecov](https://codecov.io/gh/t-900-a/moneroodoo/branch/master/graph/badge.svg)](https://codecov.io/gh/t-900-a/moneroodoo)
+
```
Comment (serhack) [inline monero-rpc-odoo/README.md:8]: A suggestion for you: add CREDITS part, well done! :)

File: monero-rpc-odoo/README.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+# Monero Odoo
+
+Allows you to accept Monero as Payment within your Odoo Ecommerce shop
```
Comment (serhack) [inline monero-rpc-odoo/README.md:10]: A link to Odoo project?

**Review by serhack — APPROVED** [review]

**Pull request merged**